### PR TITLE
initial commit.  Created a basic webpack.config.js file, some basic .…

### PR DESCRIPTION
**Pull request by rarebreed**
…babelrc settings, and some notes on the build

Here's a basic webpack 2 build setup.  It has some babel settings already and the package.json has a few things installed as dev dependencies, mainly webpack itself and some babel node modules.

I left the directory structure mostly free.  I just assumed a src/components folder for now (assuming this is where react components would live).  The entry point for webpack to build the dependency graph is src/index.js.

### CODE REVIEW

**Comment by mvollmer** [comment]
Fyi, I have done something similar here: https://github.com/mvollmer/cockpit-app-freeipa.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "subscription-manager",
+  "version": "0.1.0",
+  "description": "a plugin for cockpit to manage subscriptions",
+  "main": "index.js",
+  "repository": "git@github.com:rarebreed/subscription-manager.git",
```
Comment (rarebreed) [inline package.json]: Should change this to git@github.com/cockpit-project/subscription-manager.git

**Comment by dperpeet** [comment]
Are there plans for yarn to land in fedora? Until this is the case, it's a bit troublesome to add the repo - and there's the question of maintenance.

**Comment by dperpeet** [comment]
I have a work in progress branch here: https://github.com/dperpeet/subscription-manager/tree/build-fixes

**Comment by dperpeet** [comment]
Replaced by #2

**Pull request closed without merging**